components/contribution-flow/StepDetails.js

```diff
@@ -127,6 +127,17 @@ const StepDetails = ({ onChange, data, collective, tier, showPlatformTip, router
                 prependProps={{ color: 'black.500' }}
                 required
                 onChange={value => {
+                  const previousValue = data?.amount;
+                  // Increase/Decrease the amount by 0.5 instead of 0.01 when using the arrows
+                  const isTopArrowClicked = value - previousValue === 1;
+                  const isBottomArrowClicked = value - previousValue === -1;
+
+                  if (isTopArrowClicked) {
+                    value = value + 49;
+                  } else if (isBottomArrowClicked) {
+                    value = value - 49;
+                  }
```

Comment:
This `49` looks a bit like dark magic. Did you consider preventing the original event (`preventDefault`) to completely control the value?

Alternatively, it would be nice to add a comment for it.

Reply:
> Did you consider preventing the original event (preventDefault) to completely control the value?

Tried after this suggestion, didn't seem to work as expected. Added a comment instead. Another alternative is to use `value = previousValue + 50`

Reply:
Ideally, the 0.5 number would be easily tweaked. If we decide to switch to 1 increment tomorrow, let it be a single variable change.

Reply:
@znarf refactored 👍🏻

Reply:
As per https://github.com/opencollective/opencollective/issues/6381#issuecomment-1411100288, this will ultimately become a prop variable.